eclass/toolchain.eclass

```diff
@@ -1207,6 +1207,12 @@ toolchain_src_configure() {
 		is-flagq -mfloat-gprs=double && confgcc+=( --enable-e500-double )
 		[[ ${CTARGET//_/-} == *-e500v2-* ]] && confgcc+=( --enable-e500-double )
 		;;
+	ppc64)
+		# On ppc64 big endian target gcc assumes elfv1 by default,
+		# and elfv2 on little endian
+		# but musl does not support elfv1 at all on any endian ppc64
```

Comment:
Please add the link directly to the comment line around workaround. Otherwise it's ok to push to `::gentoo`.

Reply:
thanks!